src/Common/Markdown/MarkDown.tsx

```diff
@@ -0,0 +1,100 @@
+import { marked } from 'marked'
+import DOMPurify from 'dompurify'
+import { useEffect, useRef } from 'react'
+
+const MarkDown = (
+    setExpandableIcon: (arg0: boolean) => void,
+    markdown = '',
+    className = '',
+    breaks = false,
+    disableEscapedText = false,
+    ...props: any[]
+) => {
+    const uncheckedCheckboxInputElement = `<input checked="" disabled="" type="checkbox">`
+    const checkedCheckboxInputElement = `<input disabled="" type="checkbox">`
+    const renderer = new marked.Renderer()
+    const mdeRef = useRef(null)
+
+    const getHeight = () => {
+        const editorHeight = mdeRef.current?.clientHeight
+        const minHeight = 320
+        const showExpandableViewIcon = editorHeight > minHeight
+        if (typeof setExpandableIcon === 'function') {
+            setExpandableIcon(showExpandableViewIcon)
+        }
+    }
+    useEffect(() => {
+        getHeight()
+    }, [markdown])
+
+    function isReadmeInputCheckbox(text: string) {
+        if (text.includes(uncheckedCheckboxInputElement) || text.includes(checkedCheckboxInputElement)) {
+            return true
+        }
+        return false
+    }
```

Comment:
Also, we can use arrow function

```suggestion
    function isReadmeInputCheckbox(text: string) {
        return (text.includes(uncheckedCheckboxInputElement) || text.includes(checkedCheckboxInputElement))
    }
```

Reply:
.